packages/expo-web-browser/ios/EXWebBrowser/EXWebBrowser.m

```diff
@@ -2,7 +2,10 @@
 
 #import <SafariServices/SafariServices.h>
 #import <EXWebBrowser/EXWebBrowser.h>
+
 #import <UMCore/UMUtilities.h>
+#import "UIColor+Extension.h"
```

Comment:
I think this one color-decoding method could be just added to this class (maybe as a static method?)

Moreover, next time use `#import <LibraryName/HeaderFile.h>`, otherwise files with the same names will conflict. 😕